frontend/src/routes/_dashboard/create-project.lazy.tsx

```diff
@@ -1,11 +1,17 @@
 import * as Form from "@radix-ui/react-form";
-import { createFileRoute } from "@tanstack/react-router";
+import { useMutation } from "@tanstack/react-query";
+import { createFileRoute, useNavigate } from "@tanstack/react-router";
+import { AlertCircle } from "lucide-react";
+import { type RequestInput, apiClient } from "~/api/client";
 import { withAuthRedirect } from "~/components/helper/auth-redirect";
 import { useAuthUser } from "~/components/helper/use-auth-user";
 import { MetaTitle } from "~/components/meta-title";
+import { Alert, AlertDescription, AlertTitle } from "~/components/ui/alert";
 import { Button } from "~/components/ui/button";
 import { Input } from "~/components/ui/input";
 import { Textarea } from "~/components/ui/textarea";
+import { getFormErrorsFromResponseData } from "~/lib/utils";
+import { getCookie, getCsrf } from "~/utils";
```

Comment:
```suggestion
import { getCsrf } from "~/utils";
```

You can remove `getCookie` as it is not used anymore